nss-day-cohort-31/celebrity-tribute-picture-perfect
# added DS_Store to gitignore and created main.css

Added .DS_Store to .gitignore
Added main.css

Steps to test:
1: `git fetch --all`
2: `git checkout bm-gitignore-main.css`
3: run 'http-server'
4: Verify that .gitignore has .DS_Store and main.css created
